FlashSpace/Workspaces/WorkspaceManager.swift

```diff
@@ -258,4 +270,14 @@ extension WorkspaceManager {
             .first { NSMouseInRect(cursorLocation, $0.frame, false) }?
             .localizedName
     }
+
+    private var cursorScreenWorkspaces: (screen: DisplayName?, workspaces: [Workspace]) {
+        let screen = getCursorScreen()
+        let hasMoreScreens = NSScreen.screens.count > 1
+        return (
+            screen,
+            workspaceRepository.workspaces
+                .filter { !hasMoreScreens || $0.display == screen }
+        )
+    }
```

Comment:
Change `getCursorScreen()` to be called from outside,
Why don't you change it to a form like `workspaces(in cursorScreen: DisplayName) -> [Workspace])`?

Reply:
You are right, probably this way would be better. You can change it if you wish.

Reply:
fixed it